examples/libbacktrace/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c)
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/cpp-pm/gate
+include("../common.cmake")
+
+project(libbacktrace)
```

Comment:
I think previous examples are called download-foo
in your case download-libbacktrace

Reply:
Ok fixed.